README.md

```diff
@@ -1,34 +1,32 @@
 # vue-jest
 
-Jest transformer for Vue single file components
+Jest transformer for Vue Single File Components.
 
-> **NOTE:** This is documentation for `vue-jest@4.x`. [View the vue-jest@3.x documentation](https://github.com/vuejs/vue-jest/tree/v3)
+## Installation
 
-## Usage
+Since we need to support a variety of Vue and Jest versions, vue-jest don't follow semantic versioning.
 
-```bash
-npm install --save-dev vue-jest
-yarn add vue-jest --dev
-```
-
-### Usage with Babel 7
+|Vue version|Jest Version|Package|
+|-----------|------------|-------|
+|Vue 2|Jest 26 and below|`vue-jest@4`|
```

Comment:
You could add that Vue 3 w/ Jest 26 is `vue-jest@5`

Reply:
```suggestion
|Vue 2|Jest 26 and below|`vue-jest@4`|
|Vue 3|Jest 26 and below|`vue-jest@5`|
```